package.json

```diff
@@ -17,7 +17,7 @@
     "format:js": "npx prettier \"packages/**/*.{ts,js}\" \"crates/rspack_plugin_runtime/**/*.{ts,js}\" --check --write",
     "format-ci:toml": "npx @taplo/cli format --check '.cargo/*.toml' './crates/**/Cargo.toml' './Cargo.toml'",
     "format:toml": "npx @taplo/cli format '.cargo/*.toml' './crates/**/Cargo.toml' './Cargo.toml'",
-    "lint:js": "npx prettier \"packages/**/*.{ts,js}\" --check",
+    "lint:js": "oxlint .",
```

Comment:
it seems oxclint only lint code and not format code，so maybe we should keep prettier check？